requirements/test.txt

```diff
@@ -5,3 +5,4 @@ pytest-optional-tests >= 0.1.1
 pytest-cov
 coveralls
 pytorch-memlab
+einops
```

Comment:
This should be in `requirements.txt` as it is required by users to run `backpack`; not only to execute the tests